data/sql/updates/pending_db_world/rev_1696128697046487685.sql

```diff
@@ -0,0 +1,41 @@
+-- Move Mindless Skeletons (11197) created by Baron Rivendare (10440)
+-- to safe spots where they will not fall through the floor.
+DELETE FROM `spell_target_position` WHERE `ID`=17475 AND `EffectIndex`=0;
+INSERT INTO `spell_target_position` (`ID`, `EffectIndex`, `MapID`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `VerifiedBuild`) VALUES (17475, 0, 329, 4050.39, -3363.61, 115.05, 2.48, 0);
+DELETE FROM `spell_target_position` WHERE `ID`=17476 AND `EffectIndex`=0;
+INSERT INTO `spell_target_position` (`ID`, `EffectIndex`, `MapID`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `VerifiedBuild`) VALUES (17476, 0, 329, 4052.56, -3350.93, 115.05, 3.11, 0);
+DELETE FROM `spell_target_position` WHERE `ID`=17477 AND `EffectIndex`=0;
+INSERT INTO `spell_target_position` (`ID`, `EffectIndex`, `MapID`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `VerifiedBuild`) VALUES (17477, 0, 329, 4048.98, -3339.65, 115.06, 3.74, 0);
+DELETE FROM `spell_target_position` WHERE `ID`=17478 AND `EffectIndex`=0;
+INSERT INTO `spell_target_position` (`ID`, `EffectIndex`, `MapID`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `VerifiedBuild`) VALUES (17478, 0, 329, 4016.50, -3340.00, 115.06, 5.71, 0);
+DELETE FROM `spell_target_position` WHERE `ID`=17479 AND `EffectIndex`=0;
+INSERT INTO `spell_target_position` (`ID`, `EffectIndex`, `MapID`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `VerifiedBuild`) VALUES (17479, 0, 329, 4012.95, -3352.47, 115.05, 0.10, 0);
+DELETE FROM `spell_target_position` WHERE `ID`=17480 AND `EffectIndex`=0;
+INSERT INTO `spell_target_position` (`ID`, `EffectIndex`, `MapID`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `VerifiedBuild`) VALUES (17480, 0, 329, 4015.36, -3363.66, 115.06, 0.66, 0);
+
+-- Additionally, cause the Baron to despawn his summoned Mindless Skeletons
+-- when he is reset.
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 10440;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 10440);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(10440, 0, 0, 0, 0, 0, 100, 0, 4000, 9000, 7000, 11000, 0, 0, 11, 15284, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Every 7 - 11 seconds (4 - 9s initially) - Self: Cast spell 15284 on Victim'),
```

Comment:
Change the comments to the more modern standard "Baron Rivendare - IC - Cast full spell name"